bigchaindb/common/transaction.py

```diff
@@ -508,20 +503,20 @@ def validate_asset(self, amount=None):
                                   ' greater than one')
 
 
-class AssetLink(Asset):
+class AssetLink(object):
```

Comment:
Agreed, but let's move this to another PR after #962 is merged so that we can clean up both.